pom.xml

```diff
@@ -229,21 +259,115 @@
                     </plugin>
                 </plugins>
             </build>
-        </profile>
+        </profile> -->
 
         <profile>
-            <id>glassfish-managed</id>
+            <id>weld-se</id>
             <activation>
-                <activeByDefault>true</activeByDefault>
+                <activeByDefault>false</activeByDefault>
             </activation>
             <dependencies>
-                <dependency>
+               <!--<dependency>
                     <groupId>org.jboss.arquillian.container</groupId>
                     <artifactId>arquillian-glassfish-managed-3.1</artifactId>
-                    <version>1.0.0.CR4</version>
+                    <version>1.0.0.Final</version>
+                    <scope>test</scope>
+                </dependency> -->
+		<dependency>
+			<groupId>org.jboss.arquillian.container</groupId>
+			<artifactId>container-se-managed</artifactId>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.jboss.weld.se</groupId>
+			<artifactId>weld-se-shaded</artifactId>
+			<scope>test</scope>
+		</dependency>
+
+                <dependency>
+                    <groupId>org.jboss.arquillian.protocol</groupId>
+                    <artifactId>arquillian-protocol-servlet</artifactId>
+                    <version>${arquillian.version}</version>
                     <scope>test</scope>
                 </dependency>
                 <dependency>
+                    <groupId>log4j</groupId>
+                    <artifactId>log4j</artifactId>
+                </dependency>
+
+            </dependencies>
+            <build>
+                <resources>
+                    <resource>
+                        <directory>src/test/resources</directory>
+                    </resource>
+                    <resource>
+                        <directory>src/test/se</directory>
+                    </resource>
+                </resources>
+                <plugins>
+		<plugin>
+		    <groupId>org.apache.maven.plugins</groupId>
+		    <artifactId>maven-dependency-plugin</artifactId>
+		        <executions>
+				<execution>
+					<id>copy-cdi-dependencies</id>
+					<phase>test-compile</phase>
+						<goals>
+						<goal>copy-dependencies</goal>
+						</goals>
+					<configuration>
+			<stripVersion>true</stripVersion>
+			<overWriteReleases>true</overWriteReleases>
+			<outputDirectory>${project.build.outputDirectory}</outputDirectory>
+			<includeGroupIds>org.jboss.weld.se</includeGroupIds>
+			</configuration>
+			</execution>
+			</executions>
+			</plugin>
+
+
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <version>2.10</version>
+                        <configuration>
+                            <suiteXmlFiles>
+                                <suiteXmlFile>src/test/tck12/tck-tests.xml</suiteXmlFile>
```

Comment:
Yes, but this is the canonical exclude list. It would make sense to introduce new branch for CDI 1.2 (as @mkouba suggested) and keep stuff for 2.0 in master. Otherwise  you can create copy of this canonical exclude list e.g in `src/test/resources` and keep the exclude list for 1.2 in the `tck12` directory.